prometheus/src/chart.stories.tsx

```diff
@@ -0,0 +1,363 @@
+          [1698322202.024, '0.012135432652576494'],
+          [1698322204.024, '0.012135432652576494'],
+          [1698322206.024, '0.012135432652576494'],
+          [1698322208.024, '0.012135432652576494'],
+          [1698322210.024, '0.012135432652576494'],
+          [1698322212.024, '0.012135432652576494'],
+          [1698322214.024, '0.012135432652576494'],
+          [1698322224.024, '0.012269806297630569'],
+          [1698322226.024, '0.012269806297630569'],
+          [1698322228.024, '0.012269806297630569'],
+          [1698322230.024, '0.012269806297630569'],
+          [1698322232.024, '0.012269806297630569'],
+          [1698322234.024, '0.012269806297630569'],
+          [1698322236.024, '0.012269806297630569'],
+          [1698322238.024, '0.012269806297630569'],
+          [1698322240.024, '0.012269806297630569'],
+          [1698322242.024, '0.012269806297630569'],
+          [1698322244.024, '0.012269806297630569'],
+          [1698322246.024, '0.012269806297630569'],
+          [1698322248.024, '0.012269806297630569'],
+          [1698322254.024, '0.010410530344099453'],
+          [1698322256.024, '0.010410530344099453'],
+          [1698322258.024, '0.010410530344099453'],
+          [1698322260.024, '0.010410530344099453'],
+          [1698322262.024, '0.010410530344099453'],
+          [1698322264.024, '0.010410530344099453'],
+          [1698322266.024, '0.010410530344099453'],
+          [1698322268.024, '0.010410530344099453'],
+          [1698322270.024, '0.010410530344099453'],
+          [1698322272.024, '0.010410530344099453'],
+          [1698322274.024, '0.010410530344099453'],
+          [1698322276.024, '0.010410530344099453'],
+          [1698322278.024, '0.010410530344099453'],
+          [1698322280.024, '0.010410530344099453'],
+          [1698322282.024, '0.010410530344099453'],
+        ],
+      },
+    ],
+  },
+};
+
+const XTickProps = {
+  dataKey: 'timestamp',
+};
+
+const YTickProps = {
+  domain: ['dataMin', 'auto'],
+};
+
+function dataProcessor(response: any): { timestamp: number; y: number }[] {
+  const data: { timestamp: number; y: number }[] = [];
+  // convert the response to a JSON object
+  response['data']['result'][0]['values'].forEach(element => {
+    // convert value to a number
+
+    data.push({ timestamp: element[0], y: Number(element[1]) });
+  });
+  return data;
+}
+
+export default {
+  title: 'Chart',
+  component: Chart,
+  decorators: [
+    Story => {
+      return (
+        <BrowserRouter>
+          <Story />
+        </BrowserRouter>
+      );
+    },
+  ],
+} as Meta;
+
+// fetchMock.config.fallbackToNetwork = true;
+
+const Template: Story<ChartProps> = () => {
+  //   const mockData = data =>
+  //     fetchMock.get('*', () => {
+  //       return data;
+  //     });
+
+  //   React.useEffect(() => {
+  //     mockData(data);
+  //     return () => {
+  //       fetchMock.restore(); // Cleanup the mock when the component unmounts
+  //     };
+  //   }, []);
```

Comment:
Should this be deleted? If it's here to help with debug/dev, we should add a comment indicating that.